source/templates.tex

```diff
@@ -7881,8 +7881,8 @@
 Each deduction deduces template arguments for subsequent positions in
 the template parameter packs expanded by the function parameter pack.
 When a function parameter pack appears in a non-deduced
-context\iref{temp.deduct.type}, the type of that pack is
-never deduced.
+context\iref{temp.deduct.type}, the template parameter pack(s)
+expanded by that pack are never deduced, even if they appear in a deduced context elsewhere.
```

Comment:
The new wording seems confusing: packs are expanded by pack expansions, not by packs. Replacing "that pack" with "that function parameter pack" would help. Also, "(s)" -> "s", please.

Is it clear that the "even if" part is the intent? This seems ambiguous.

Reply:
... not only ambiguous, but inconsistent with unanimous implementation practice: https://godbolt.org/z/78TjoY

Reply:
IMO, it seems the current wording is suitable for either case. 
> If a template parameter is used only in non-deduced contexts and is not explicitly specified, template argument deduction fails.  
>
> When a function parameter pack appears in a non-deduced context ([temp.deduct.type]), the type of that pack is never deduced.  

1. Template parameter that appears in non-deduced context does not participate in template argument deduction 
2. These template parameters that **only** appear in non-deduced context should be explicitly specified their template arguments, otherwise, the deduction fails. 

In addition, except that [temp.deduct.call#1] explicitly says that
> When a function parameter pack appears in a non-deduced context ([temp.deduct.type]), the type of that pack is **never deduced**.

It seems that the other kinds of argument deduction do not strong this concept, that is  
> any template parameter that appears in a non-deduced context is never deduced